### PR TITLE
Split Navigation and Search pages

### LINKED ISSUE #1047: Split navigation and search docs in user guide
This page is now VERY VERY LONG, I think it will be more readable/scannable if we have separate pages for navigation and search.

https://www.docsy.dev/docs/adding-content/navigation/

### PR DESCRIPTION
**Pull request by LisaFC**
Fixes #1047

---

**Preview**: https://deploy-preview-1352--docsydocs.netlify.app/docs/adding-content/search/

### CODE REVIEW

**Comment by LisaFC** [comment]
Work so far splits the pages and fixes the couple of internal links that the split broke. There *is* also the possibility of it breaking external links, though sadly I'm pretty sure you can't redirect to/from anchors....

File: userguide/content/en/docs/adding-content/navigation.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 ---
-title: Navigation and Search
+title: "Navigation and Menus"
```
Comment (chalin) [inline userguide/content/en/docs/adding-content/navigation.md:2]:
```suggestion
title: Navigation and Menus
```

File: userguide/content/en/docs/adding-content/navigation.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 date: 2017-01-05
 weight: 3
-description: Customize site navigation and search for your Docsy site.
+description: >
+   Customize site navigation for your Docsy site.
```
Comment (chalin) [inline userguide/content/en/docs/adding-content/navigation.md:6]:
```suggestion
description: Customize site navigation for your Docsy site.
```

File: userguide/content/en/docs/adding-content/search.md [hunk 3]
```diff
@@ -0,0 +1,279 @@
+---
+title: "Search"
```
Comment (chalin) [inline userguide/content/en/docs/adding-content/search.md:2]:
```suggestion
title: Search
```

**Review by chalin — COMMENTED** [review]
Thanks for working on this. See inline for a few nits.

**Comment by LisaFC** [comment]
Don't think there's anything else I need to do here so I am un-WIP-ing it. @chalin @geriom @emckean want to take a look?

**Comment by chalin** [comment]
LGTM aside from the nits identified above. The link checker seems happy, so that's good. (The issue of title font color on the homepage is fixed via #1467, FYI.)

**Review by deining — APPROVED** [review]
LGTM

**Review by chalin — CHANGES_REQUESTED** [review]
The new Search page is missing changes that were applied to the "Site search options" section since this PR was created: the most obvious are the updates to the config file names, but there could be more. @LisaFC - did you recopy the "Site search options" section from `@HEAD`?

**Comment by LisaFC** [comment]
I thought I did but possibly it got lost in the merge. Let me add those in again. Thanks for spotting!

**Comment by LisaFC** [comment]
Hang on, I've realized they're in the intro, which I didn't change because I rewrote it. Let me fix the file names.

**Review by chalin — APPROVED** [review]
LGTM, thanks!

**Pull request merged**